iam.tf

```diff
@@ -35,6 +35,7 @@ resource "aws_iam_policy" "ecs-policy" {
   name_prefix = "${replace(format("%.102s", replace("tf-ECSInPol-${var.name}-", "_", "-")), "/\\s/", "-")}"
   description = "A terraform created policy for ECS"
   path        = "${var.iam_path}"
+  count       = "${1 - var.use_custom_iam_policy}"
```

Comment:
this is correct, but IMO counterintuitive.  a more common idiom is the ternary operator, like this:

```hcl
count = "${var.use_custom_iam_policy ? 1 : 0}"
```

in this case, however, you don't even need the boolean; try something like this:

```hcl
count = "${length(var.custom_iam_policy) > 0 ? 1 : 0}"
```

and then you make the default value of `custom_iam_policy` the empty string.

does that make sense?

Reply:
glad you like the idea. I've tried to use ternary operator at first but haven't figured out the length function, which will probably make all the difference :)
I'm trying the changes now and will update the PR